src/snovault/elasticsearch/indexer.py

```diff
@@ -204,12 +207,14 @@ def _send_sqs_msg(
         try: 
             queue_arn = sqs_resource.get_queue_by_name(QueueName='ec2_scheduler_input')
             if queue_arn:
+                print(queue_arn.url)
                 response = sqs_resource.meta.client.send_message(
                     QueueUrl=queue_arn.url,
                     DelaySeconds=delay_seconds,
                     MessageAttributes=msg_attrs,
                     MessageBody=body,
                 )
+                print(response)
```

Comment:
is this  a debug print?

Reply:
removed, looked for more but didn't find any